conversion_sr14_sr15/config.yaml

```diff
@@ -0,0 +1,17 @@
+# Input settings
+input:
+  input_dir: '/home/eamcevoy/edna.aguilar/OneDrive - Resource Systems Group, Inc/Documents/20_Projects/SANDAG/inputs'
+  filenames:
+    households: 'households.csv'
+    persons: 'persons.csv'
+    mgra_xwalk: 'xref_mgra13-15.csv'
```

Comment:
Is this crosswalk file same format as the one that I shared before? If not, could you share it?

Reply:
It is the same format as the one you provided

Reply:
How about crosswalk for TAZ? The households file has the column TAZ, and it was not converted from SR14 to SR15.